src/04_features/filter/filter.scss

```diff
@@ -0,0 +1,51 @@
+
+.filter{
+    &__subject{
+        margin-bottom: 14px;
+        width: 194px;
+        height: 44px;
+        padding: 8px 16px 8px 18px;
+        border-radius: 10px;
+        background: rgb(var(--clr-primary-grey));
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+    
+    &__text{
+        margin: 0;
+        font-weight: 400;
+        font-size: 16px;
+        line-height: 150%;
+        text-align: center;
+        color: rgb(var(--clr-white));
+    }
+    &__img{
+        width:14px;
+        height: 12px;
+        transition: color 0.3s ease-in-out;
+    } 
+    &__img:focus{
+        color:#9747ff;
+    }  
+    &__img:hover{
+        color:#8937f4;
+    } 
+    &__img:active{
+        color:#802af1;
+    } 
+    &__search{
+        width: 194px;
+        height: 44px;
```

Comment:
do not use fconstant height and width. stretch layout
for exp:
wodth: 100%;
max-width: 200px;